pkg/providers/vsphere/vmlifecycle/update_status_test.go

```diff
@@ -2756,7 +2756,7 @@ var _ = Describe("Hardware status", func() {
 				vmCtx.MoVM.Config = &vimtypes.VirtualMachineConfigInfo{
 					Hardware: vimtypes.VirtualHardware{
 						NumCPU:   4,
-						MemoryMB: 2048,
+						MemoryMB: 4096,
```

Comment:
Drop a comment here indicating that you are choosing this value so that the int32 boundary of 2,147,483,647 is crossed?

Reply:
good point. I added a new UT with a proper description.